std/datetime.d

```diff
@@ -16171,6 +16179,14 @@ public:
         mixin(format(`return retval._addSeconds(convert!("hnsecs", "seconds")(%shnsecs));`, op));
     }
 
+    ///
+    unittest
+    {
+        DateTime oldYear = DateTime(2015, 12, 31, 23, 59, 59);
+        DateTime newYear = oldYear + 1.seconds;
+        assert(newYear == DateTime(2016, 1, 1, 0, 0, 0));
```

Comment:
I know I'm being OCD, but the examples should be consistent -- one uses "auto", the other doesn't, one calls the variables "st1" and "st2", the other calls them "oldYear" and "newYear"

Other than that, LGTM.